components/uploader.py

```diff
@@ -0,0 +1,57 @@
+import dash_html_components as html
+import dash_core_components as dcc
+import dash_table
+from utils import dataparser
+
+
+def get_div():
```

Comment:
Maybe get_component() would be a better name? I think it would be logical if all components had a get_component() function, then you wouldn't have to always look at the component code to find out the function name. I don't think we can assume that all components are wrapped inside a Div, or can we?

Reply:
Good point, I've now changed it